README.md

```diff
@@ -38,33 +47,58 @@ Required and compatibility dependencies will be automatically installed by pip.
 -   [plumbum](https://github.com/tomerfiliba/plumbum): Command line tools
 -   [lark-parser](https://github.com/lark-parser/lark): A modern parsing library for Python
 
-### Python compatibility:
+#### Python compatibility:
 -   [six](https://github.com/benjaminp/six): Compatibility library
 -   [pathlib2](https://github.com/mcmtroffaes/pathlib2) backport if using Python 2.7
 -   [enum34](https://bitbucket.org/stoneleaf/enum34) backport if using Python /< 3.5
 -   [importlib_resources](http://importlib-resources.readthedocs.io/en/latest/) backport if using Python /< 3.7
 
 
-### Recommended dependencies:
+#### Recommended dependencies:
 -   [graphviz](https://gitlab.com/graphviz/graphviz/) to render (DOT
```

Comment:
And pydot?

Reply:
I omitted it for now as we have so much as work-in-progress. But can add if you prefer? OK, will do in a sec.